elrond-wasm-debug/src/world_mock/blockchain_mock.rs

```diff
@@ -18,6 +13,7 @@ const ELROND_REWARD_KEY: &[u8] = b"ELRONDreward";
 #[derive(Debug)]
 pub struct BlockchainMock {
     pub accounts: HashMap<Address, AccountData>,
+    pub addr_to_mandos_string_map: HashMap<Address, String>,
```

Comment:
We could wrap this into an object, for future extensions, but for now it looks great